nmrfx-processor/src/main/java/org/nmrfx/processor/processing/processes/ProcessOps.java

```diff
@@ -339,13 +313,8 @@ public Object callMatrix() {
                 }
             }
 
-            if (error) {
-                break;
-            }
+            vectorsProcessed++;
 
-            if (matrix != null) {
-                vectorsProcessed++;
-            }
```

Comment:
removed if statement, only issue could be if evalMatrix set matrix to null, looked through the evalMatrix methods, didn't seem like any of them did that, so removed it